### DISCUSSION TITLE
Can I set additional commands after installation?

### DISCUSSION THREAD

**Discussion by wonkyungup** [Homebrew/discussions#3811] [Casks] [answered]
After typing brew install

Is there any possible way to install the program and add the path (environment)?

**Comment by SMillerDev** [comment, marked as answer]
You can make a function that does extra things like
```sh
function brew()
{
  brew $1
  echo "brewed $1"
}
```

**Reply by wonkyungup** [reply]
@SMillerDev thank you for the reply.

Is it possible to add it to the rb file?

**Reply by SMillerDev** [reply]
What rb file? brew.rb? You'll have to make a pull request for that.

**Reply by wonkyungup** [reply]
sorry. The file I was talking about was brew.rb. Is there any way to run echo like above in brew.rb?

**Reply by SMillerDev** [reply]
If you maintain your own version of brew, yes.

**Reply by wonkyungup** [reply]
@SMillerDev 
Oh, I see.

Thank you for your kind reply.
